instrumentation/aws-sdk/aws-sdk-1.11/javaagent/src/test/groovy/SnsTracingTest.groovy

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import static io.opentelemetry.api.trace.SpanKind.CLIENT
+import static io.opentelemetry.api.trace.SpanKind.CONSUMER
+
+import com.amazonaws.services.sns.AmazonSNSAsyncClient
+import com.amazonaws.services.sns.model.CreateTopicResult
+import com.amazonaws.services.sqs.AmazonSQSAsyncClient
+import com.amazonaws.services.sqs.model.CreateQueueResult
+import com.amazonaws.services.sqs.model.GetQueueAttributesRequest
+import com.amazonaws.services.sqs.model.ReceiveMessageRequest
+import io.opentelemetry.instrumentation.test.AgentInstrumentationSpecification
+import org.slf4j.LoggerFactory
+import org.testcontainers.containers.localstack.LocalStackContainer
+import org.testcontainers.containers.output.Slf4jLogConsumer
+import org.testcontainers.utility.DockerImageName
+import spock.lang.Ignore
+import spock.lang.Shared
+
+class SnsTracingTest extends AgentInstrumentationSpecification {
+
+  @Shared
+  int sqsPort
+  @Shared
+  LocalStackContainer localstack
+  @Shared
+  AmazonSQSAsyncClient sqsClient
+
+  @Shared
+  int snsPort
+  @Shared
+  LocalStackContainer sns
```

Comment:
Is this set anywhere?

Reply:
Good catch, it's from previous direc emq. Cleaned that up along with the rest.